dbt/include/global_project/macros/materializations/table/table.sql

```diff
@@ -48,8 +59,8 @@
   {% if non_destructive_mode -%}
     -- noop
   {%- else -%}
-    {{ drop_if_exists(existing, schema, identifier) }}
-    {{ adapter.rename(schema, tmp_identifier, identifier) }}
+    {{ drop_relation_if_exists(target_relation) }}
```

Comment:
i think this should use `old_relation`. If the thing previously existed as a view, this will run a `drop table ...` statement which causes an error